### ISSUE TITLE
Unreadable colors

### ISSUE THREAD

**Issue by bound-variable** [bigH/git-fuzzy#48]
![20220413_17h03m05s_grim](https://user-images.githubusercontent.com/51193876/163288986-92f79e77-54a2-482b-a7bb-d32a1769b8a4.png)

I've made my own colorblind colorscheme for the terminal. It's still a work in progress.

As it is, my black and white colors are identical to my default background and foreground colors respectively. I could possibly change this so that the black or the white or both are instead a variation of gray. But I can't set any more colors (including grayscale) because I can't differentiate between them.

In other words, I could possibly gain one more color definition (gray), but that's it.

Any solutions to this? Would be nice to customize the colors from within the app.

**Comment by bigH** [comment]
Fixed in 31512bc.

Try pulling, then `GRAY="$(tput setaf 7)" git fuzzy`.

**Comment by bound-variable** [comment]
Thanks
